live/src/agama-installer.kiwi

```diff
@@ -31,17 +31,17 @@
     </preferences>
     <!-- the ISO Volume ID is set by the fix_bootconfig script -->
     <preferences arch="ppc64le" profiles="openSUSE,SUSE-SLE-16,Leap-16.0">
-        <type image="iso" flags="dmsquash"  firmware="ofw" mediacheck="true" editbootconfig="fix_bootconfig">
+        <type image="iso" flags="dmsquash"  firmware="ofw" mediacheck="true" publisher="SUSE LLC" editbootconfig="fix_bootconfig">
```

Comment:
Um, do we want to use it also for the openSUSE images? The official openSUSE Leap 15.6 images use the "SUSE LINUX GmbH" publisher...